#!/usr/bin/python3

'''
raise_exception:raises exceptio
'''


def raise_exception():
    raise TypeError("Exception raised")
